File: C++/Sorting/Bubble.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int arr[]={9,8,7,6,5,5,4,3,2,1,0};
    int sizes = sizeof(arr)/sizeof(arr[0]);
    for(int i=0;i<sizes;i++){
        for(int j=0;j<sizes-i-1;j++){
            if(arr[j]>arr[j+1]){
                swap(arr[j],arr[j+1]);
            }
        }
    }

for(auto it:arr) cout<<it<<" ";
}

File: JS/Sorting/bubble.js
let arr=[7,5,2,34,5,6,78,12,45,1]

for(a in arr) console.log(a,arr[a])
console.log()
console.log()
console.log()
for(let i=0;i<arr.length-1;i++){
    for(let j=0;j<arr.length-i-1;j++){
        if(arr[j]>arr[j+1]){
            [arr[j],arr[j+1]]=[arr[j+1],arr[j]]
        }
    }
}


for(a in arr) console.log(a,arr[a])